### PR TITLE
feat(wallet-import): change passphrase size when paste

### PR DESCRIPTION
**Pull request by tibi77**
Change the size of the pass phrase if we know it is larger. We can suggest it as well if need be, but changing it looks pretty slick.

### CODE REVIEW

**Comment by tibi77** [comment]
@LuizAsFight -> IDK how to add you as a reviewer

File: packages/app/src/systems/Core/components/Mnemonic/Mnemonic.tsx [hunk 1]
```diff
@@ -152,7 +161,7 @@ export function Mnemonic({
             variant="ghost"
             color="gray"
             leftIcon={<Icon icon="ClipboardText" color="gray8" />}
-            onPress={handlePast}
+            onPress={handlePasteButton}
```
Comment (tibi77) [inline packages/app/src/systems/Core/components/Mnemonic/Mnemonic.tsx:164]: maybe change to handlePastePress ... not sure how the naming conventions work in this repo

**Comment by LuizAsFight** [comment]
hey @tibi77 all good? thanks for the contribution. two notes: 

- should add a changeset
run `pnpm changeset` in repo, then follow instructions and commit file generated


- name of PR should follow conventional commits pattern (because it will turn in a commit later)

**Comment by tibi77** [comment]
@LuizAsFight all good here; 
thanks for the pointers, I was going by the github tags that were in the issue

File: packages/app/src/systems/Core/components/Mnemonic/Mnemonic.tsx (hunk 1 above)
Comment (tomiiide) [reply packages/app/src/systems/Core/components/Mnemonic/Mnemonic.tsx:164]: yeah, `handlePastePress` seems more appropriate.

**Comment by tibi77** [comment]
@tomiiide totally agree. Added the test as a e2e test

**Comment by LuizAsFight** [comment]
@luizstacio need some eyes here I need to see "Preview" version but deployment vercel is not working (at least I try to authorize and nothing happens )

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

**Comment by LuizAsFight** [comment]
@tibi77 is this done? when it is you can re-ask review clicking here:

image

**Comment by tibi77** [comment]
yeah sorry for that @LuizAsFight 
Quick question I can see a test `crx/crx.test.ts:53:7 › FuelWallet Extension › SDK operations` keep failing in different PRs 
could someone help me understand why is that ?

**Comment by LuizAsFight** [comment]
> yeah sorry for that @LuizAsFight Quick question I can see a test `crx/crx.test.ts:53:7 › FuelWallet Extension › SDK operations` keep failing in different PRs could someone help me understand why is that ?

it's and intermittent problem someone needs to dig on it and explore the reason. it's tracked here [FRO-131](https://linear.app/fuel-network/issue/FRO-131/intermittent-test-breaking-in-e2e-tests-ci)

**Comment by tibi77** [comment]
@LuizAsFight I don t have access to that linear link

**Comment by tibi77** [comment]
@LuizAsFight kind reminder on this Review

**Comment by helciofranco** [comment]
This has been released on https://github.com/FuelLabs/fuels-wallet/pull/1099

**Pull request closed without merging**